### PR TITLE
[Mat] Added a default temperature to fluid properties

### PR DESCRIPTION
**Pull request by wenqing**
As titled.

### CODE REVIEW

File: FEM/rf_mfp_new.cpp [hunk 1]
```diff
@@ -1053,7 +1053,19 @@ void CFluidProperties::CalPrimaryVariable(
         // MX  m_pcs = PCSGet("HEAT_TRANSPORT");
         m_pcs = PCSGet(pcs_name_vector[i], true);
         if (!m_pcs)
-            return;  // MX
+        {
+            if (pcs_name_vector[i].find("TEMPERATURE1") != std::string::npos)
```
Comment (norihiro-w) [inline FEM/rf_mfp_new.cpp]: why do you compare a process name with TEMPERATURE1? or is just the variable name confusing me?

Comment (wenqing) [reply FEM/rf_mfp_new.cpp]: @norihiro-w This name is misleading. I just added another commit to change this name. Thanks for your comment.

Comment (norihiro-w) [reply FEM/rf_mfp_new.cpp]: thank you for the change!

**Comment by norihiro-w** [comment]
:+1

**Pull request merged**